deeprankcore/dataset.py

```diff
@@ -534,10 +723,16 @@ def load_one_graph(self, fname: str, entry_name: str)  -> Data: # pylint: disabl
                         vals = grp[f"{Efeat.EDGE}/{feat}"][()]
                         if vals.ndim == 1:
                             vals = vals.reshape(-1, 1)
+                            if self._standardize:
+                                vals = (vals-self.means[feat])/self.devs[feat]
+                        else:
+                            if self._standardize:
+                                reshaped_mean = [mean_value for mean_key, mean_value in self.means.items() if feat in mean_key]
+                                reshaped_dev = [dev_value for dev_key, dev_value in self.devs.items() if feat in dev_key]
+                                vals = (vals - reshaped_mean)/reshaped_dev
```

Comment:
Since edges are mirrored we need to apply the transformation separately for edges and nodes, otherwise for edges we're not going to get the right values. Additionally, `data` variable is a pytorch geom `Data` object which separates x and edge_attr, so I am not sure about how to apply standardization all together to it (we need to be careful to subtract the right mean and dev for each feature).

Reply:
OK!